.gitignore

```diff
@@ -0,0 +1,2 @@
+target/
+.idea/compiler.xml
```

Comment:
Could you havre this as the whole .idea folder rather than just compiler.xml @sauliuskucevicius ?

Reply:
Yes - good spot! I will work on those changes now

Reply:
@matthewmccormick I have made those changes in a commit now, can you review again?

Reply:
@matthewmccormick i agree as this helps the various configuration files

Reply:
Sorry for the delay, thanks for reviewing 👍